.github/workflows/ubuntu.yml

```diff
@@ -2,72 +2,40 @@ name: Ubuntu
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
   workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    
-    env: 
-      VK_VERSION: "1.2.176.1"
-      INSTALL_LOCATION: "$GITHUB_WORKSPACE/../vulkan_sdk"
 
     steps:
-
       - uses: actions/checkout@v2
-      
-      - name: update apt 
-        run: sudo apt-get update
-        
+
+      - name: update apt
+        run: |
+          sudo apt-get update
+          sudo apt-get install git build-essential libx11-xcb-dev \
+          libxkbcommon-dev libwayland-dev libxrandr-dev
```

Comment:
what are `libxkbcommon-dev`, `libwayland-dev`, and `libxrandr-dev`?

Reply:
These are GUI libraries that are needed by Linux when building the Vulkan Loader.